factorio

```diff
@@ -32,11 +32,13 @@ if [ -z "${FCONF}" ]; then
   echo "FCONF config missing! Add it to your config file"
   exit 1
 fi
-if ! [ -e ${FCONF} ]; then
```

Comment:
You are still trying to pullrequest into another branch than master which probably explains these removed lines? =)